src/app/components/layout/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import Navbar from '../navbar/navbar';
+
+export default function Layout() {
+  return (
+    <>
+      <Navbar
+        label={{
+          lblHome: 'Dallas Software Developers',
+          lblCommunity: 'Community Impact',
+          lblContact: 'Contact Us',
+          lblMeetup: 'Meetups',
+        }}
+      />
+      <main></main>
+    </>
```

Comment:
Good day @GetBlackBoxSolutions! Please I need to know if this component should return a `<div>` as the root element or is it alright this: `<>{ children }</>`